src/lua/InputApi.cpp

```diff
@@ -236,11 +236,7 @@ int LuaContext::input_api_is_mouse_button_released(lua_State* l) {
 int LuaContext::input_api_get_mouse_position(lua_State* l) {
 
   return LuaTools::exception_boundary_handle(l, [&] {
-    Point mouse_xy;
```

Comment:
If I change the Video::window_to_quest_coordinates behavior like you requested, I have to change the behavior of this function as well.
How do you want to do otherwise ?